depends/packages/qt.mk

```diff
@@ -23,7 +23,8 @@ define $(package)_set_vars
 $(package)_config_opts_release = -release
 $(package)_config_opts_debug = -debug
 $(package)_config_opts += -bindir $(build_prefix)/bin
-$(package)_config_opts += -c++std c++11
+# Note: C++17 not supported yet.
+$(package)_config_opts += -c++std c++14
```

Comment:
I'm not 100% sure on this but this answer says that it should work at Qt 5.12:
https://stackoverflow.com/questions/46610996/cant-use-c17-features-using-g-7-2-in-qtcreator

Not sure if all distros support Qt 5.12